### ISSUE TITLE
Pagination component is broken.

### ISSUE THREAD

**Issue by ericol** [IBM/carbon-components-svelte#178]
Example:

![image](https://user-images.githubusercontent.com/16800/81443297-8fd69180-914b-11ea-96e0-526bbb39fc60.png)
 How to reproduce:
```svelte
<script>
    import { Pagination } from "carbon-components-svelte";
</script>

<Pagination pageSizes={ [ ] } page={ 1 } pageSize={ 5 } totalItems={ 50 } />
```

I think the problem is here:
https://github.com/IBM/carbon-components-svelte/blob/7119feb4adf43756a4f24576c0e404d79a795976/src/components/Pagination/Pagination.svelte#L74

Because the binded value in the `Select` component is `selected` and not `defaultValue`

The page works after changing the select, but only because of this function (That I don't understand why it's here if there's a binding):

https://github.com/IBM/carbon-components-svelte/blob/7119feb4adf43756a4f24576c0e404d79a795976/src/components/Pagination/Pagination.svelte#L71-L73

The problem here is that the first time the `on:change` event is fired (Automatically), the closure function is passed null as value for `detail`. When the user changes the value of the `select` the correct value is passed to the closure.

if I change the binding to `bind:select={ page }` it works correctly (I also removed the `on:change` closure because, again, I have no idea why it's there)

The same error can be observed here:

https://github.com/IBM/carbon-components-svelte/blob/7119feb4adf43756a4f24576c0e404d79a795976/src/components/Pagination/Pagination.svelte#L50

**Comment by ericol** [comment]
P.S.: Thank you very much for such an incredible library.

Wouldn't I be such a mediocre coder I'd be sending PRs your way already (I hacked the Table component so it displays HTML and other Svelte components in cells ;) )

**Comment by metonym** [comment]
Thanks for raising this issue, @ericol . I especially appreciate the detail in debugging the problem. Would you like to submit a PR?

**Comment by ericol** [comment]
@metonym I'd love to send a PR. But not sure what to do with this:

https://github.com/IBM/carbon-components-svelte/blob/7119feb4adf43756a4f24576c0e404d79a795976/src/components/Pagination/Pagination.svelte#L71-L73

In my opinion it shouldn't be there... but again, may be it has a purpose.

I'll cook something regarding the `DataTable` (Please take into account it's **very** ugly :P )

**Comment by ericol** [comment]
@metonym I submitted #179 that fixes this issue.

**Comment by metonym** [comment]
Thanks again @ericol for the fix!

Released in v0.6.1 – [changelog notes](https://github.com/IBM/carbon-components-svelte/blob/master/CHANGELOG.md#061---2020-05-08)
